frontend/public/components/overview/hpa-overview.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { HorizontalPodAutoscalerKind } from '../../module/k8s';
+import { ResourceLink, SidebarSectionHeading } from '../utils';
+import { HorizontalPodAutoscalerModel } from '../../models';
+
+type HPAOverviewProps = {
+  hpas?: HorizontalPodAutoscalerKind[];
+};
+
+export const HPAOverview: React.FC<HPAOverviewProps> = ({ hpas }) => {
+  if (!hpas?.length) {
```

Comment:
```suggestion
  if (hpas?.length === 0) {
```

Reply:
`hpas` being null won't match `0` absolutely. So that changes the logic. Either we don't have hpas OR we have an array of zero is the logic today, if we wanted to keep that and compare against 0:
```suggestion
  if (!hpas || hpas.length === 0) {
```